lib/Serverless.js

```diff
@@ -64,6 +63,22 @@ class Serverless {
   }
 
   /**
+   * fill in the credentials by profile or by given credentials
+   */
+  _setCredentials() {
+      // Set Admin API Keys
+      var profiles = awsMisc.profilesMap();
+
+      if (!!process.env.SERVERLESS_ADMIN_AWS_PROFILE && !!profiles[process.env.SERVERLESS_ADMIN_AWS_PROFILE]) {
```

Comment:
You don't need `!!`, since `&&` (and `if` as well) converts it's operands to boolean anyway.